### PR TITLE
Added iOS courses in hindi

### PR DESCRIPTION
**Pull request by PuneetTri**
## What does this PR do?
Added resources for iOS development for beginners, and people with some CS background

### Description
Complete course + crash course on iOS development

### Why is this valuable (or not)?
Gives you a nice overview of how to work with iOS devices, and develop apps for various use cases. 

### How do we know it's really free?
It's available on youtube for free!

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by PuneetTri** [comment]
Alphabetical order is correct, its just is iOS instead of being IOS

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -260,6 +261,12 @@
 * [Tailwind CSS Tutorials in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ahwFDuExCpPFHAK829Wto2O) - CodeWithHarry
 
 
+### iOS
+
+* [iOS Development (Crash course)](https://www.youtube.com/watch?v=n7qOGHWunUY) - Akash Padhiyar
+* [iOS Development Course For Beginners](https://www.youtube.com/playlist?list=PLtCBuHKmdxOcmrDx2pM4qNvzWF2NI_Qlo) - Fahad Hussain
```
Comment (shrn01) [inline courses/free-courses-hi.md:267]:
```suggestion
* [IOS Mobile App Development Tutorial for beginners](https://www.youtube.com/playlist?list=PLtCBuHKmdxOcmrDx2pM4qNvzWF2NI_Qlo) - Fahad Hussain
```
Better to put the starting title

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -260,6 +261,12 @@
 * [Tailwind CSS Tutorials in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ahwFDuExCpPFHAK829Wto2O) - CodeWithHarry
 
 
+### iOS
+
+* [iOS Development (Crash course)](https://www.youtube.com/watch?v=n7qOGHWunUY) - Akash Padhiyar
```
Comment (shrn01) [inline courses/free-courses-hi.md:266]:
```suggestion
* [iOS App Development Crash Course in Hindi](https://www.youtube.com/watch?v=n7qOGHWunUY) - Akash Padhiyar
```
Dont change the resource name

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**